src/status_im/ui/screens/wallet/accounts/views.cljs

```diff
@@ -71,15 +71,19 @@
 (defn render-asset [currency]
   (fn [{:keys [icon decimals amount color value] :as token}]
     [list-item/list-item
-     {:content [react/view {:style {:margin-horizontal 16 :justify-content :center :flex-shrink 1}}
-                [react/view {:flex-direction :row}
-                 [react/text {:style {:font-weight "500" :flex-shrink 0.5} :number-of-lines 1 :ellipsize-mode :tail}
-                  (wallet.utils/format-amount amount decimals)]
-                 [react/text {:style           {:font-weight "500" :color colors/gray :margin-left 6}
-                              :number-of-lines 1}
-                  (wallet.utils/display-symbol token)]]
-                (when value
-                  [react/text {:style {:color colors/gray}} (str value " " currency)])]
+     {:title-prefix (wallet.utils/format-amount amount decimals)
+      :title        (wallet.utils/display-symbol token)
+      :title-color-override colors/gray
+      :subtitle (str (if value value 0) " " currency)
+      ;; :content [react/view
```

Comment:
same as above please add a comment in the code on why it is commented out and a TODO about when/why uncomment it

Reply:
got it 👍 will rectify

Reply:
done